File: Programming Language(ver.2)/Python/601.py
input, n = input(), int(input())
output = ''
if input[n].isupper():
    output = input[:n] + input[n].lower() + input[n+1:]
else:
    output = input[:n] + input[n].upper() + input[n+1:]
print('The letter that was selected is: ' + output[n])
print(output)

File: Programming Language(ver.2)/C#/601.cs
using System;

public class TQC601
{
    public static void Main(string[] args)
    {
        string data = Console.ReadLine();
        int n = int.Parse(Console.ReadLine());
        char[] dataChars = data.ToCharArray();
        if (n>=0 && n<data.Length)
        {
            char item =  dataChars[n];
            if (item >= 'a' && item <= 'z')
            {
                dataChars[n] = Char.ToUpper(item);
            }
            else if (item >= 'A' && item <= 'Z')
            {
                dataChars[n] = Char.ToLower(item);
            }        
        }
        string newData = new string(dataChars);
        Console.WriteLine($"The letter that was selected is: {dataChars[n]}");
        Console.WriteLine(newData);
    }
}